_posts/2022-11-02-relicensing.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License][GPLv2] (GPL v2+) to the more permissive [Lesser GNU
+Lesser Public License][LGPL] (LGPL v3+) license. Both are *open source
+licenses* but the [LGPL][] gives developers more freedom in how they
+license any of their own code that makes use of MDAnalysis.
+
+The license is the legal basis that determines how code can be copied,
+changed, and incorporated into other code. The license is the only
+reason that one can use someone else's code --- it does not matter if
+the code is publicly available somewhere. If it does not have a
+license you cannot use it legally. Therefore, getting the license
+right is very important.
+
+**In order to change the MDAnalysis license, we need the agreement of
+all our contributors to change from GPL to LGPL.**
+
+In this post we want to share the motivation, outline the relicensing
+process, and invite questions from the community.
+
+
+
+## Why not GPL any more?
+
+Since its initial release in 2008, MDAnalysis has grown from a small
+Python package used by a handful of enthusiastic graduate students and
+postdocs to a mature library that is used by thousands of researchers
+in the molecular sciences. The MDAnalysis library was published under
+an open source license from the start so that anyone could freely use
+it, contribute to it, and build on it. We chose the GNU Public License
+(GPL) for this purpose. Thanks to the "copy-left" portion of the GPL,
+which requires anyone using MDAnalysis in their own code to also adopt
+the GPL for their code, contributors could feel that any time and work
+that they donated and invested into MDAnalysis would not end up
+contributing to software that they could not use themselves in the
+future.
+
+However, the GPL has also created barriers to adoption of MDAnalysis
+in that it prevented developers to make their own code available under
+other permissive open source licenses. Ultimately, we do not want to
+tell our developers and users how to license their code but we also
```

Comment:
todo: reword as our own legal interpretation, we don't want to be getting into a fight with folks out there that believe a different interpretation

Reply:
For everything that we are saying we should have a disclaimer that this our interpretation. However, I have no problem defending our interpretation.

Reply:
> I have no problem defending our interpretation.

Generally INAL should stand here, we have taken advice from a lawyer and we are sticking with it. I would always advise making sure that language is as soft as possible in order to not have to suddenly take a stance as a non-legal professional.